scrubble-client/src/html/game.html

```diff
@@ -1,5 +1,61 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Scrubble</title>
+  </head>
   <body>
-    <div id="pixi-application"></div>
+    <header class="header">
+      <p class="header__logo">Scrubble</p>
+    </header>
+    <section class="room-code">
+      <h2 class="room-code__label">Room Code:</h2>
+      <h2 class="room-code__code">WXYZ</h2>
+    </section>
+    <section class="players">
+      <div class="players__current-turn" id="players__1">
+        <h3 class="players__name" id="players__1-name">Player 1:</h3>
+        <h3 class="players__score" id="players__1-score">24</h3>
+      </div>
+      <div class="players__player" id="players__2">
```

Comment:
This player score element seems like exactly the kind of thing we should look into using webcomponents for at some point (just an observation, no change needed here)

Reply:
gotchya sounds good. yeah this seems like it will be clunky if executed as is